src/components/Datepicker/components/DatepickerDays.vue

```diff
@@ -89,6 +112,25 @@ export default {
       const _day = dayJsInstance.day()
       return _day === 0 ? 7 : _day
     },
+
+    isDisabledDay(dateValue) {
+      let disabled = false
+      if (this.disabledPast && dayjs().isAfter(dateValue, 'day')) {
+        disabled = true
+      } else if (
+        this.minDate &&
+        dayjs(dateValue).isBefore(this.minDate, 'day')
+      ) {
+        disabled = true
+      } else if (
+        this.maxDate &&
+        dayjs(dateValue).isAfter(this.maxDate, 'day')
+      ) {
+        disabled = true
+      }
+
+      return disabled
+    },
```

Comment:
The same thing as above. Or, does it make sense to use here the computed property from the `Datepicker` component?

Reply:
@emanuelgsouza It's the same thing, I just did it like this to reduce the complexity of the code.